CMakeLists.txt

```diff
@@ -28,6 +35,13 @@ daq_codegen(
 			 timingfanoutcontroller.jsonnet 
 			 DEP_PKGS appfwk rcif cmdlib iomanager TEMPLATES Structs.hpp.j2 Nljs.hpp.j2 )
 
+daq_add_python_bindings(dal_methods.cpp module.cpp LINK_LIBRARIES 
```

Comment:
I don't see the source you are referring to here in a pybind11 directory and the generate_modules_test program belongs to the appdal package. I suspect you have cut and paste too much from there.